cross/squid/PLIST

```diff
@@ -1,75 +1,36 @@
-bin:sbin/squid
-etc:etc/cachemgr.conf
-etc:etc/cachemgr.conf.default
-etc:etc/errorpage.css
-etc:etc/errorpage.css.default
-etc:etc/mime.conf
-etc:etc/mime.conf.default
-etc:etc/msntauth.conf
-etc:etc/msntauth.conf.default
-etc:etc/squid.conf
-etc:etc/squid.conf.default
-etc:etc/squid.conf.documented
+bin:bin/purge
+bin:bin/squidclient
+rsc:libexec/basic_db_auth
+bin:libexec/basic_fake_auth
+bin:libexec/basic_getpwnam_auth
+bin:libexec/basic_ldap_auth
+rsc:libexec/basic_msnt_multi_domain_auth
```

Comment:
See my earlier comment, I'd suggest to use the appropriate prefix regardless if a file can actually be stripped.